crates/bevy_window/src/event.rs

```diff
@@ -171,6 +172,7 @@ pub struct CursorLeft {
 }
 
 /// An event that is sent whenever a window receives a character from the OS or underlying system.
+#[deprecated = "Use `KeyboardInput` instead."]
```

Comment:
```suggestion
#[deprecated(
    since = "0.14.0",
    note = "Use `KeyboardInput` instead."
)]
```
adding the version is useful when we need to delete these later.